app/schema.py

```diff
@@ -128,6 +128,40 @@ class Feedback(BaseModel):
     feedback: Optional[constr(max_length=2000)]
 
 
-class Resource(BaseModel):
-    name: constr(max_length=255)
-    item_id: constr(max_length=255)
+class TicketsTable(BaseModel):
+    ticket_id: constr(max_length=255)
+    ticket_type: Optional[Literal['Action', 'Application', 'Resource', 'Role']]
+    ticket_status: Optional[Literal['Pending', 'Approved', 'Rejected']]
+    ticket_subject: constr(max_length=255)
+    urgent: Optional[Literal['Low', 'Normal', 'High']]
+    notes: constr(max_length=255)
+    requested_for: constr(max_length=255)
+    submitted_by: constr(max_length=255)
+    approved_by: constr(max_length=255)
+
+
+class Assignment(BaseModel):
+    mentor_id: constr(max_length=255)
+    mentee_id: constr(max_length=255)
+    assignment_id: constr(max_length=255)
+
+
+class Resources(BaseModel):
+    resource_id: constr(max_length=255)
+    updated_at: constr(max_length=255)
+    resource_name: constr(max_length=255)
+    category: constr(max_length=255)
+    condition: constr(max_length=255)
+    assigned: constr(max_length=255)
+    current_assignee: constr(max_length=255)
+    previous_assignee: constr(max_length=255)
+    monetary_value: constr(max_length=255)
+    deductible_donation: constr(max_length=255)
```

Comment:
string doesn't seem like the right type for `deductible_donation`—maybe a boolean (is the donation deductible or not) or a float (the donation value that can be deducted). We should confirm with FE the purpose of this field.

Reply:
Good point, will confirm with FE before committing any changes!